Configuration/StandardSequences/python/AlCaHarvesting_cff.py

```diff
@@ -217,15 +189,13 @@
 BeamSpotByLumi = cms.Path(ALCAHARVESTBeamSpotByLumi)
 BeamSpotHPByRun  = cms.Path(ALCAHARVESTBeamSpotHPByRun)
 BeamSpotHPByLumi = cms.Path(ALCAHARVESTBeamSpotHPByLumi)
-BeamSpotHPLowPUByRun  = cms.Path(ALCAHARVESTBeamSpotHPLowPUByRun)
-BeamSpotHPLowPUByLumi = cms.Path(ALCAHARVESTBeamSpotHPLowPUByLumi)
```

Comment:
same comment here

Reply:
@fabiocos it is an accident. I will fix that. Thanks very much for pointing it. Sorry I didn't check the different wrt to the mater branch.